src/router/summary_router.py

```
@@ -0,0 +1,15 @@
+from src.service.summary_service import SummaryService, get_summary_meeting
+from enum import Enum
+
+router = APIRouter()
+
+from src.env_variable import MEETING_TEXT
+
+with open(MEETING_TEXT, encoding='utf-8') as f:
+         meeting_text = f.read()
+
+@router.post('/summary')
+def summary(summary_service : SummaryService = Depends(get_summary_meeting)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)